README.md

```diff
@@ -1,4 +1,8 @@
 Citadel
 =======
 
-A turn based strategy game based on the Anura engine
\ No newline at end of file
+A turn based strategy game based on the Anura engine
+
+For support, to meet the player community and find worthy opponents for multiplayer, drop by the Discord server 
+[![Discord Server Registration](https://img.shields.io/badge/discord-server_registration-green.svg?longCache=true&style=popout&logo=discord)](https://discord.gg/Y5e3phv)
+[![Discord Server](https://img.shields.io/badge/discord-server-blue.svg?longCache=true&style=popout&logo=discord)](https://discordapp.com/channels/156241487938584577/)
```

Comment:
I think I get the point of showing two links instead of just one, but is this second link really worth it?

Reply:
It depends? If you have a discord account and have already joined, but aren't logged in, having a separate link that will just prompt you to log in is easier. If we just keep the join link, people might accidentally join the server with multiple accounts, because the "Already have an account" link is very small.

My use case might be marginal and I'm ok with removing the second link